src/plugins/intel_cpu/src/nodes/executors/fullyconnected_config.hpp

```diff
@@ -18,8 +17,6 @@ struct FCAttrs {
     // @todo probably we don't want with bias flag, since this information is already
     // a part of src memory descs
     bool withBias = false;
-    // @todo probably we don't want with bias flag, since this information is already
-    // a part of weight memory desc
     bool weightsNonTransposed = false;
     bool sparseWeights = false;
     // @todo only memory descriptors should be a part of attributes
```

Comment:
What's stopping you from doing this? As far as I understand it, we agreed on passing all the memory parameters via the corresponding input and output maps of memory objects.

Reply:
Hm, then yes, it's better to leave the transpose flag and other properties that can't be retrieved from descriptors.